keyform/views.py

```diff
@@ -9,10 +9,39 @@
 from django.http import HttpResponseRedirect
 from django.contrib.auth.mixins import LoginRequiredMixin
 
-class HomeView(LoginRequiredMixin, ListView):
+
+class HomeView(ListView):
     model = Request
     template_name = "keyform/home.html"
 
+    def get_context_data(self):
+        context = super(HomeView, self).get_context_data()
+        context["request_types"] = Request.REQUEST_TYPES
+        context["status_types"] = Request.STATUS_TYPES
+        return context
+
+
+class SearchView(ListView):
```

Comment:
Dearest Patrick,

It's been a fortnight since I've spoken to you last. The moon has changed phases, the sun grows ever hotter, I had to get my air conditioning fixed. While I still haven't mastered the ability to drive, I grow ever closer to finally, FINALLY putting the keys in the ignition without screaming like the day I was born. 

ResNet has been sorely lacking without you. I miss your broken watch, your Captain America backpack, and your login screen just scares into my soul with no hope of relief. Really, what world have you left me to in your absence? What could you possibly be doing in that wide, expansive, wonderful world that we have all abandoned in order to live in a basement? 

Anyways, I feel it is important that you understand that LoginRequiredMixin is actually very important, and you seem to have deleted it in these two classes. I am quite flummoxed as to why you'd have done this, was it to spite me personally? Have I wronged you in some way? You see, without this, we lose a lot of our security. All of it, as a matter of fact, as anyone could access any of these pages without logging in. 

I hope to hear from you soon, and that you are successful in all of your future endeavors. 

Sincerely yours, 

Rei Ozymandias

P.S. Guy came in here looking for you. Real Grim Reaper-type. I don’t know. Might further the plot.